app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -346,9 +346,10 @@ angular.module('Bastion.components').factory('Nutupane',
                 if (!column) {
                     return;
                 }
-
-                params["sort_by"] = column.id;
-                if (column.id === sort.by) {
+                if (column.id) {
+                    params["sort_by"] = column.id;
+                }
+                if (column.id === sort.by || column.id === "") {
```

Comment:
Why the check for an empty string for `column.id`?   Do we want to check for more than empty string?  What if `column.id` is `null` or `undefined`?